core/domain/stats_domain.py

```diff
@@ -403,6 +405,215 @@ def validate(self):
                     '%s cannot have negative values' % (stat_property))
 
 
+class ExplorationIssues(object):
+    """Domain object representing the exploration to issues mapping for an
+    exploration.
+    """
+
+    def __init__(self, exp_id, unresolved_issues):
+        """Constructs an ExplorationIssues domain object.
+
+        Args:
+            exp_id: str. ID of the exploration.
+            unresolved_issues: list(dict). The unresolved issues for this
```

Comment:
If possible, enumerate what this dict would look like. Example - https://github.com/oppia/oppia/blob/develop/core/domain/classifier_domain.py#L54

Reply:
Done.